<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IConfigurationDialog</class>
 <widget class="QDialog" name="IConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TuxCards - Configuration</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabGeneral">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Save File</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QCheckBox" name="mpAutoSaveCheckBox">
              <property name="text">
               <string>Automatically  save all</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mpSaveMinutesLineEdit">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>15</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Minutes</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="mpSaveWhenQuittingCheckBox">
            <property name="text">
             <string>Save File when quitting TuxCards</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mpCreateBackupFileCheckBox">
            <property name="text">
             <string>Create Backup File</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mpExecuteCommandGroupBox">
         <property name="title">
          <string>Execute Command after Saving</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="mpCommandLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Select Icon Directory for Look &amp;&amp; Feel</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="mpIconDirectoryLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Encyption</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QCheckBox" name="mpAutoEncryptCheckBox">
            <property name="text">
             <string>Automatically encrypt Entry when selecting another one</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>SideBar</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QWidget" name="mpPlaceHolder" native="true">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_10">
           <property name="title">
            <string>Choose Colors</string>
           </property>
           <layout class="QGridLayout">
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Top Color</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="mpTopColorButton">
              <property name="toolTip">
               <string>Changes the side bar's top color.</string>
              </property>
              <property name="text">
               <string>Top Color</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Bottom Color</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="mpBottomColorButton">
              <property name="toolTip">
               <string>Changes the side bar's bottom color.</string>
              </property>
              <property name="text">
               <string>Bottom Color</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Text Color</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="mpTextColorButton">
              <property name="toolTip">
               <string>Changes the side bar's text color.</string>
              </property>
              <property name="text">
               <string>Text Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mpHorizontalTextEnabled">
           <property name="title">
            <string>Show Horizontal Text</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>First Text Line</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="mpFirstTextLineLineEdit">
              <property name="text">
               <string>Tux</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Second Text Line</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="mpSecondTextLineLineEdit">
              <property name="text">
               <string>Cards</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mpVerticalTextEnabled">
           <property name="title">
            <string>Show Vertical Text</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLineEdit" name="mpVerticalTextLineLineEdit">
              <property name="text">
               <string>TuxCards</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Text Position</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="mpAlineTop">
                <property name="text">
                 <string>Top</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="mpAlineBottom">
                <property name="text">
                 <string>Bottom</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mpEnableCactusBar">
           <property name="title">
            <string>Use CactusBar</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Flower Directory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mpFlowerDirectoryLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string notr="true">/usr/local/bin/flowers</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_2">
      <attribute name="title">
       <string>Tree</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Select Font for Tree</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QPushButton" name="mpChangeTreeFontButton">
            <property name="text">
             <string>Change Tree Font</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mpTreeFontExample">
            <property name="text">
             <string>Tree_FontExample</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_3">
      <attribute name="title">
       <string>Editor</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="toolTip">
          <string>The selected font is used for text in ASCII format and as default font for text in RTF format.</string>
         </property>
         <property name="title">
          <string>Select Font</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QPushButton" name="mpChangeEditorFontButton">
            <property name="text">
             <string>Change Editor Font</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mpEditorFontExample">
            <property name="text">
             <string>EditorFont_Example</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Tab Spacing</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Set Tabsize to</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mpTabSizeLineEdit">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>WordWrap</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QRadioButton" name="mpNoWordWrap">
            <property name="text">
             <string>no WordWrap</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QRadioButton" name="mpColumnWrap">
              <property name="text">
               <string>Wrap at Column</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mpWrapAtColumnLineEdit">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="mpWindowWidthWrap">
            <property name="text">
             <string>Wrap at Window-Width</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mpLineBreakGroupBox">
         <property name="title">
          <string>Line Break Mode</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QRadioButton" name="mpEnableLineBreakWithCtrlShift">
            <property name="text">
             <string>Single LineBreak with CRTL/SHIFT + ENTER/RETURN</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mpEnableLineBreakWithoutCtrlShift">
            <property name="text">
             <string>Single LineBreak with ENTER/RETURN</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEnableCactusBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>mpFlowerDirectoryLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpExecuteCommandGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mpCommandLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_8</sender>
   <signal>toggled(bool)</signal>
   <receiver>mpIconDirectoryLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEnableCactusBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_7</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
